### PR TITLE
Issue 70

### PR DESCRIPTION
**Pull request by aeciopires**
#### What this PR does / why we need it:

*  Hotfix about initContainers and add support .Chart.appVersion
*  Bump chart version
*  Added notice about Zabbix version installed by default
*  Change Zabbix version installed by default

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

**Comment by aeciopires** [comment]
Hi @sa-ChristianAnton!

I fixed the problems reported in PR #67.

Could do you a review today?

Thanks by your help

**Review by sa-ChristianAnton — APPROVED** [review]
looks exactly as I would have done it 👍 Haven't tested it in real life though.

File: values.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@
 # Declare variables to be passed into your templates.
 
 # -- zabbix components (server, agent, web frontend, ...) image tag to use. Overwritten by zabbixserver.image.tag etc.
-zabbix_image_tag: ubuntu-6.2.0
+zabbix_image_tag: ubuntu-6.0.7
```
Comment (sa-ChristianAnton) [inline values.yaml:6]: This should be left commented by default, as now the tag defaults to AppVersion, right?

File: Chart.yaml [hunk 2]
```diff
@@ -1,8 +1,9 @@
 ---
 apiVersion: v2
 name: zabbix
-version: 3.1.0
-appVersion: 6.2.0
+version: 3.1.1
+# LTS Zabbix version by default due to stability. See: https://www.zabbix.com/life_cycle_and_release_policy
+appVersion: 6.0.0
```
Comment (sa-ChristianAnton) [inline Chart.yaml:6]: here we should have the latest LTS: 6.0.7

File: values.yaml (hunk 1 above)
Comment (aeciopires) [reply values.yaml:6]: Make sense, @sa-ChristianAnton! I will comment.

File: Chart.yaml (hunk 2 above)
Comment (aeciopires) [reply Chart.yaml:6]: I agree. Doing...

Comment (aeciopires) [reply Chart.yaml:6]: @sa-ChristianAnton using the .Chart.appVersion don't works for Zabbix tags:
https://hub.docker.com/r/zabbix/zabbix-server-mysql/tags?page=1&name=6.0.7

In this case, the name of tag has distroname + version number. 

The .Chart.appVersion not support distroname. I will remove the .Chart.appVersion of templates. Do you agree?

File: values.yaml (hunk 1 above)
Comment (aeciopires) [reply values.yaml:6]: @sa-ChristianAnton using the .Chart.appVersion don't works for Zabbix tags:
https://hub.docker.com/r/zabbix/zabbix-server-mysql/tags?page=1&name=6.0.7

In this case, the name of tag has distroname + version number.

The .Chart.appVersion not support distroname. I will remove the .Chart.appVersion of templates. Do you agree?

File: Chart.yaml (hunk 2 above)
Comment (sa-ChristianAnton) [reply Chart.yaml:6]: Too bad, didn't think about this. Sorry. Back to before then....

Comment (aeciopires) [reply Chart.yaml:6]: Technically the concatenation of the names is good challenge, but I think that cause the confusing for other people when they need  change the version and this aggregate few value for helm chart. Do you agree?

Comment (sa-ChristianAnton) [reply Chart.yaml:6]: I think you may be right. let's then go back to how it originally was.

Comment (aeciopires) [reply Chart.yaml:6]: Thank you for your understanding. Doing...

**Review by sa-ChristianAnton — APPROVED** [review]

**Comment by aeciopires** [comment]
Thanks @sa-ChristianAnton for review and contribution.

**Pull request merged**